src/libAtomVM/mailbox.c

```diff
@@ -353,8 +361,9 @@ void mailbox_crashdump(Context *ctx)
     ctx->mailbox.outer_first = mailbox_process_outer_list(&ctx->mailbox);
     MailboxMessage *msg = ctx->mailbox.inner_first;
     while (msg) {
-        term_display(stderr, msg->body.normal.message, ctx);
+        struct Message *normal_message = CONTAINER_OF(msg, Message, base);
```

Comment:
It seems to be yet another inconsistent use of `Message` and `struct Message` on the same line.

Reply:
I replaced all `struct Message` with `Message`.